sonic-thermalctld/scripts/thermalctld

```diff
@@ -378,12 +391,12 @@ class TemperatureUpdater(object):
         Update all temperature information to database
         :return:
         """
-        logger.log_debug("Start temperature updating")
-        try:
-            for index, thermal in enumerate(self.chassis.get_all_thermals()):
+        logger.log_debug("Start temperature updating") 
+        for index, thermal in enumerate(self.chassis.get_all_thermals()):
+            try:
                 self._refresh_temperature_status(thermal, index)
-        except Exception as e:
-            logger.log_warning('Failed to update thermal status - {}'.format(e))
+            except Exception as e:
+                logger.log_warning('Failed to update thermal status - {}'.format(e))
 
         logger.log_debug("End temperature updating")
```

Comment:
Can you remove debugging log, which shows up every 60s in syslog
```
DEBUG pmon#thermalctld: Start temperature updating
DEBUG pmon#thermalctld: End temperature updating
```
@Junchao-Mellanox

Reply:
Sure, will remove it in another PR.